safety_filter_mpc:
  ros__parameters:
    safety_filter:
      ts: 0.025
      timeout: 0.1

      collisions:
        radius: 0.4
        front_offset: 0.2
        back_offset: 0.2

      constraints:
        lbx: [-1.0, -1.5]
        ubx: [1.0, 1.5]
        lbu: [-1.0, -1.5]
        ubu: [1.0, 1.5]
        uh: [10000.0, 10000.0]
        lsh: [0.0, 0.0]
        ush: [10000.0, 10000.0]

      cost:
        W_0: [5.0, 5.0, 4.0, 4.0]
        W: [2.0, 2.0]
        Zl: [1000.0, 1000.0]
        Zu: [1000.0, 1000.0]
        zl: [800.0, 800.0]
        zu: [800.0, 800.0]